proto/accessgraph/v1alpha/access_graph_service.proto

```diff
@@ -94,6 +143,53 @@ service AccessGraphService {
 
   // NetIQEventsStream is a stream of commands to the NetIQ importer.
   rpc NetIQEventsStream(stream NetIQEventsStreamRequest) returns (stream NetIQEventsStreamResponse);
+
+  // GitHubAuditLogStream establishes a persistent bidirectional stream for
+  // exporting GitHub audit log events from a client (ex.: a Teleport connector
+  // for GitHub) to the AccessGraphService.
+  //
+  // This stream facilitates: - Initial configuration exchange (e.g., setting
```

Comment:
same comment as aws/teleport config and resume state responses are sent using two different messages